docs/products/kafka/kafka-connect/howto/debezium-source-connector-mongodb.rst

```diff
@@ -0,0 +1,100 @@
+Create a Debezium source connector for MongoDB
+==================================================
+
+The Debezium source connector for MongoDB tracks database changes using a MongoDB replica set or shared cluster, and writes them to an Apache Kafka® topic in a standard format where they can be transformed and read by multiple consumers.
+
+.. _connect_debezium_mongodb_source_prereq:
+
+Prerequisites
+-------------
+
+To setup a Debezium source connector pointing to MongoDB, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to collect the following information about the source MongoDB database upfront:
+
+* ``MONGODB_HOST``: The database hostname
+* ``MONGODB_PORT``: The database port
+* ``MONGODB_USER``: The database user to connect
+* ``MONGODB_PASSWORD``: The database password for the ``MONGODB_USER``
+* ``MONGODB_DATABASE_NAME``: The database name to include in the replica
+* ``MONGODB_REPLICA_SET_NAME``: The name of MongoDB's replica set
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    If you're using Aiven for Apache Kafka®,  the Kafka related details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup a MongoDB Debezium source connector with Aiven Console
+------------------------------------------------------------
+
+The following example demonstrates how to setup a Debezium source connector for Apache Kafka to a MongoDB database using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`.
+
+Define a Kafka Connect configuration file
+'''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``debezium_source_mongodb.json``) with the following content:
```

Comment:
Do we need to justify creating a file, since we're just going to copy its content later on?

(I personally think it's easier to assemble the information in a file first, especially if the text editor being used helps format JSON, but this does feel a bit like it might be left over from when the CLI was being used. So it might be worth saying it's easier to get the file right before pasting it into the console form.)